roles/chrony/defaults/main.yml

```diff
@@ -6,3 +6,8 @@ public_ntp_servers:
   - time2.google.com
   - time3.google.com
   - time4.google.com
+
+chrony_package: chrony
+chrony_service: chrony
+chrony_dir: /etc/chrony/
+chrony_filename: chrony.conf
```

Comment:
You're setting Debian defaults and changing them if the system is RedHat. This is a good approach to reduce duplicated code, but you can do it more generic:

- Set those variables to 'DEFAULT'
- Then if it's RedHat set RedHat values, but with `when: variable_name == 'DEFAULT'`
- Then do the same with Debian

A nice thing with this approach is that one could also set a variable to a non-default value (to support another distro, or a non common configuration) and it will not be replaced.

Consider putting this code in separate files like `debian.yml` and `redhat.yml`. So in 2 years, when you completely forgot how this role is written, if you have to add support to another system, you will find out quickly what you need to do.